site/e2e/api.ts

```diff
@@ -5,9 +5,11 @@ import { findSessionToken, randomName } from "./helpers";
 
 let currentOrgId: string;
 
-export const setupApiCalls = async (page: Page) => {
-  const token = await findSessionToken(page);
-  API.setSessionToken(token);
+export const setupApiCalls = async (page: Page, unauthenticated?: boolean) => {
```

Comment:
having a `boolean` as an optional argument is almost always a bad idea imo, because from the call site `setupApiCalls(page, true)` is incredibly vague. it should either be on an options object, or ideally imo, it would just gracefully do nothing if there is no token, and not take any additional params.

```
try {
  const token = await findSessionToken(page);
  API.setSessionToken(token);
} catch {}
```